.github/workflows/review.yml

```
@@ -21,110 +21,82 @@ jobs:
+          HEAD=${{ github.event.pull_request.head.sha }}
+          git fetch origin $BASE $HEAD --depth=1 || true
+          git diff --unified=0 $BASE $HEAD > diff.patch
+          wc -l diff.patch || true
+
+      - name: Write reviewer (CommonJS) and run
         run: |
           mkdir -p tools
           cat > tools/reviewer.js <<'JS'
-import fs from 'fs';
-// Parse args
+const fs = require('fs');
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 36: could not find expected ':'

(syntax-check)